src/lexer/lexer.c

```diff
@@ -68,10 +147,14 @@ t_list	*lexer(const char *command_line, t_exitstatus *exit_status)
 	t_list	*tokens;
 	char	*token;
 
```

Comment:
Probaly it is a good idea to add an assertion here: assert(command_line != NULL);